mlx/directives/item_relink_directive.py

```diff
@@ -33,12 +33,13 @@ def apply_effect(self, collection):
         reverse_type = collection.get_reverse_relation(forward_type)
 
         if source is None:
-            report_warning("Could not find item {!r} specified in item-relink directive".format(source_id),
-                           self['document'], self['line'])
+            report_warning("Could not find item {!r} with type {!r} specified in item-relink directive"
+                           .format(source_id, forward_type), self['document'], self['line'])
             return
         if not reverse_type:
-            report_warning("Could not find reverse relationship type for type {!r} specified in item-relink directive"
-                           .format(forward_type), self['document'], self['line'])
+            report_warning(("Could not find reverse relationship type for type {!r} specified in "
+                            "{!r} item-relink directive").format(forward_type, source_id),
```

Comment:
What's the use of this change? The source_id has no impact on the fact that the reverse relation is not defined. What am I missing?

Reply:
I just wanted to align it with warning above where I could not pinpoint which item-relink is the culprit for the warning. And adding type information helps it make unique.

Reply:
You didn't add type information in this message. You added `source_id` in an awkward location in the warning message.

Reply:
No, but type does not uniquely reference the `item-relink` so you need source_id to find which one it is. So on the end it would be: `Could not find reverse relationship type for type "passed" specified in "REMAPPED_ITEM" item-relink directive.`

So I just named the item-relink directive for easier searching.